### PR TITLE
Add docker resources for ubuntu, centos platforms

### PR DESCRIPTION
**Pull request by sajinieKavindya**
## Purpose
Create docker resources for ubuntu, centos platforms.

Related issue: https://github.com/wso2/micro-integrator/issues/165

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -220,9 +219,45 @@
...
+                        <id>build-image-centos</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <configuration>
+                            <skipDocker>${docker.skip}</skipDocker>
+                            <imageName>wso2/micro-integrator</imageName>
+                            <imageTags>
+                                <imageTag>${project.version}-centos</imageTag>
```
[the remark this answers is not in the document]
Comment (Nirothipan) [reply distribution/pom.xml]: Please check others as well .

File: distribution/pom.xml [hunk 2]
```diff
@@ -220,9 +219,45 @@
...
+                        <id>build-image-ubuntu</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <configuration>
+                            <skipDocker>${docker.skip}</skipDocker>
+                            <imageName>wso2/micro-integrator</imageName>
+                            <imageTags>
+                                <imageTag>${project.version}-ubuntu18.04</imageTag>
```
Comment (Nirothipan) [inline distribution/pom.xml]: Do we usually name the ubuntu tags ?

File: distribution/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (sajinieKavindya) [reply distribution/pom.xml]: changed as requested.

File: distribution/pom.xml (hunk 2 above)
Comment (sajinieKavindya) [reply distribution/pom.xml:258]: We will keep the alpine one as the default one and will name the ubuntu tag as in the PR.

**Pull request merged**